packages/browser/core/src/core/auth/internal/StitchAuthImpl.ts

```diff
@@ -423,7 +421,7 @@ export default class StitchAuthImpl extends CoreStitchAuth<StitchUser>
   }
 
   private cleanupRedirect() {
-    this.jsdomWindow.history.replaceState(null, "", this.pageRootUrl());
+    this.jsdomWindow.history.replaceState(undefined, "", this.pageRootUrl());
```

Comment:
Hmm this one is tricky. I can't tell from Googling if this will work. It will _probably_ work, but to be sure you should set up an E2E scenario with Google redirect auth, and see if the auth tokens are cleared from the URL after redirect auth is complete. If you need help with this lmk, I have a pretty good idea of what you need to do.